boa_interner/src/lib.rs

```diff
@@ -67,44 +67,161 @@
     clippy::missing_errors_doc,
     clippy::as_conversions,
     clippy::let_unit_value,
-    rustdoc::missing_doc_code_examples
+    // TODO deny once false positive are fixed (https://github.com/rust-lang/rust-clippy/issues/9076).
+    clippy::trait_duplication_in_bounds,
```

Comment:
This seems to be fixed already, right? At least in the clippy repo.

Reply:
Not really. If I remove that, and run `cargo clippy`, the output is:

```
error: these where clauses contain repeated elements
   --> boa_interner/src/lib.rs:194:12
    |
194 |         C: From<&'a str> + From<&'b [u16]>,
```

Reply:
Opened https://github.com/rust-lang/rust-clippy/issues/9626, will change the link to this.